frontend/src/utils/storage.ts

```diff
@@ -0,0 +1,12 @@
+export const STORAGE_KEYS = {
+  AUDIO_SOURCE: 'audioSource',
+  VIDEO_SOURCE: 'videoSource',
+};
```

Comment:
Note: we use `setItem` and `getItem` for a few other things such as background blur etc but I didn't want to include changes to that in this PR since it seems unrelated. I'll create a follow up ticket and change it there